modules/ding_wayf_dk/includes/DingWayfGateway.class.inc

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * @file
+ * Simple class for communication with the SimpleSAMLphp DBC gateway.
+ */
+
+namespace WAYF;
+use WAYF\SPortoException;
+
+/**
+ * DingWayfGateway class
+ */
+class DingWayfGateway implements WayfServiceInterface {
```

Comment:
It's defined inside the wayf_dk module as a way to ensure that the functions required in services class are implemented in the class.

It's part of the patch https://www.drupal.org/files/issues/allow_other_libraries-2648680-1.diff to the module on DO. Which is patched in the make file.